dateutil/test/test_isoparser.py

```diff
@@ -306,14 +306,14 @@ def test_isoparser_invalid_sep(sep):
 
 
 # This only fails on Python 3
-@pytest.mark.xfail(six.PY3, reason="Fails on Python 3 only")
-def test_isoparser_byte_sep():
-    dt = datetime(2017, 12, 6, 12, 30, 45)
-    dt_str = dt.isoformat(sep=str('T'))
+@pytest.mark.xfail(six.PY3, reason="Fails on Python 3 only")    #pragma: no cover
+def test_isoparser_byte_sep():                                  #pragma: no cover
```

Comment:
This whole function does not need `# pragma: nocover` because it is indeed already covered by the Python 2 tests.

Reply:
Can xfails be made strict? Makes sure we get alerted if something is accidentally fixed.

Reply:
They are already strict. It's configured globally in the pytest section of setup.cfg